docs/a320-simvars.md

```diff
@@ -407,6 +415,74 @@
     - Hz
     - The VHF 3 standby frequency for the right RMP.
 
+- A32NX_RMP_L_SAVED_ACTIVE_FREQUENCY_VOR
```

Comment:
**suggestion:** format those like, for example, the contactor open simvar documentation, instead of a lot of repetitive entries